test/whitebox_test.proto

```diff
@@ -0,0 +1,40 @@
+// WhiteBoxTest represents a list of test APIs that facilitates testing the
+// switch/controller (injecting events, mutating switch state etc.)
+
+syntax = "proto3";
+
+package gnoi.gnoi;
+
+service WhiteBoxTest {
```

Comment:
This could be achieved by configuration by configuring [/system/control-plane-traffic/](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-system.html#system-control-plane-traffic-ingress-acl-acl-set)

However, another approach might be if your use case is to completely block a gRPC service for a fixed amount of time and then allow the DUT to restore it.  You could define something like:

service SystemServices
// temporarily change the state of /system/services

message InterruptServiceState
          string service_name = 1;
          uint32 duration = 2;   // msec of interruption